pkg/vmprovider/providers/vsphere2/vmlifecycle/bootstrap.go

```diff
@@ -148,7 +157,14 @@ func getBootstrapArgs(
 			continue
 		}
 
-		if len(r.Nameservers) == 0 || (isTKG && len(r.SearchDomains) == 0) {
+		if len(bootstrapArgs.DNSServers) == 0 && len(r.Nameservers) == 0 {
+			missingDNSInfo = true
+			break
+		}
+
+		// Previously in v1a1 we would not set the global search suffixes for LinuxPrep and Sysprep
```

Comment:
Can we use some type of pattern that indicates when something is for a specific schema version? Maybe:

```go
V1ALPHA1: Do not default the global search suffixes for LinuxPrep and Sysprep
          to what is in the ConfigMap.
```